lib/plugin/instance.rb

```diff
@@ -785,7 +785,7 @@ def automatic_assets
     js = "(function(){#{js}})();" if js.present?
 
     result = []
-    result << [css, "css"] if css.present?
+    result << [css, "scss"] if css.present?
```

Comment:
These were always being interpreted as scss files in the past. But now that we're using `@import` on them instead of concatenation, the extension is important.